cgmanifest.json

```diff
@@ -49,7 +49,7 @@
          "component": {
             "type": "git",
             "git": {
-               "commitHash": "1facb4c1bb9cc2107d4dbaf9fd647fefdbbeb0ab",
+               "commitHash": "8d15705ec29c2867847d1a8affad9fe19dedb2e3",
```

Comment:
Could you please split the updating of ONNX into a separated PR? Because often it's problematic.

Reply:
@codemzs would it be possible to update ONNX in a separate PR to master?

Reply:
@snnn Can you please clarify why is it problematic? I would want to assess potential risk before we invest anytime in separating out the PR. We need this change for subsequent PRs that are in the pipeline.

Reply:
ONNX 1.7 is not released yet. In general, ORT won't take an arbitrary commit from ONNX master. I can make an exception this time.

But usually the effort of advancing the ONNX commit is non-trivial. A lot of files need be changed, A lot of bugs need be fixed. For example, when you do it, you saw a lot tests (>20) were failed.  We should fix all the test failures found by onnx_test_runner.




---
In reply to: [392520645](https://github.com/microsoft/onnxruntime/pull/3174#discussion_r392520645) [](ancestors = 392520645)

Reply:
thanks @snn, agreed on "But usually the effort of advancing the ONNX commit is non-trivial" , @SherlockNoMad for FYI.

Reply:
it looks like there is still work to be done here then:
- pick a commit from an ONNX release branch
- fix issues, e.g., fix test failures

---
In reply to: [393369900](https://github.com/microsoft/onnxruntime/pull/3174#discussion_r393369900) [](ancestors = 393369900)

Reply:
Any updates?

Reply:
@codemzs will update ONNX to a release branch commit

---
In reply to: [398744563](https://github.com/microsoft/onnxruntime/pull/3174#discussion_r398744563) [](ancestors = 398744563)

Reply:
this has already happened in master and ort_training has the change.